style.css

```diff
@@ -602,6 +745,24 @@
     gap: 0;
   }
 
+  .detail-card-body-left {
+    width: 70%;
+  }
+
```

Comment:
On the desktop version, when I open the project detail, the popup window should be on the center of the screen with the scroll locked and the background with some `filter: blur`

- Kindly position your card detail in the middle of the screen

- Kindly add the blur filter on the background of the popup window